django/db/migrations/questioner.py

```diff
@@ -44,7 +44,6 @@ def ask_initial(self, app_label):
         except ImportError:
             return self.defaults.get("ask_initial", False)
         else:
-            # getattr() needed on PY36 and older (replace with attribute access).
```

Comment:
Thanks for not heeding the comment! I came here just to check on it... (Cf. #13841, specifically [this](https://github.com/django/django/pull/13841/commits/34424973507422b05cd6f1bf0cd51d874a46d182#diff-17c5ace9fb3dcbc1f10671d7fd7d248bc394020ce095b0c6a94c8a163a8d4a54L47))